src/InteractivityComponents/Dropdown.php

```diff
@@ -40,45 +38,46 @@ public static function render( $props ) {
 
 		ob_start();
 		?>
-	<div class="wc-block-stock-filter style-dropdown" data-wc-context='<?php echo wp_json_encode( $dropdown_context ); ?>' >
-		<div class="wc-blocks-components-form-token-field-wrapper single-selection" >
-			<div class="components-form-token-field" tabindex="-1">
-				<div class="components-form-token-field__input-container" 
-							data-wc-class--is-active="context.woocommerceDropdown.isOpen" 
-							tabindex="-1" 
-							data-wc-on--click="actions.woocommerceDropdown.toggleIsOpen" 
-					>
-					<input id="components-form-token-input-1" type="text" autocomplete="off" data-wc-bind--placeholder="selectors.woocommerceDropdown.placeholderText" class="components-form-token-field__input" role="combobox" aria-expanded="false" aria-autocomplete="list" aria-describedby="components-form-token-suggestions-howto-1" value="">
-					<ul hidden data-wc-bind--hidden="!context.woocommerceDropdown.isOpen" class="components-form-token-field__suggestions-list" id="components-form-token-suggestions-1" role="listbox">
-						<?php
-						foreach ( $items as $item ) :
-							$context = array(
-								'woocommerceDropdown' => array( 'currentItem' => $item ),
-								JSON_NUMERIC_CHECK,
-							);
-							?>
-							<li
-								role="option" 
-								data-wc-on--click--select-item="actions.woocommerceDropdown.selectDropdownItem" 
-								data-wc-on--click--parent-action="<?php echo esc_attr( $action ); ?>" 
-								data-wc-class--is-selected="selectors.woocommerceDropdown.isSelected" 
-								data-wc-on--mouseover="actions.woocommerceDropdown.addHoverClass" 
-								data-wc-on--mouseout="actions.woocommerceDropdown.removeHoverClass" 
-								data-wc-context='<?php echo wp_json_encode( $context ); ?>' 
-								class="components-form-token-field__suggestion" 
-								data-wc-bind--aria-selected="selectors.woocommerceDropdown.isSelected"
+		<div data-wc-interactive='<?php echo wp_json_encode( array( 'namespace' => 'woocommerce/interactivity-dropdown' ) ); ?>'>
```

Comment:
All blocks/inner blocks/anything with corresponding frontend store should have a namespace directive in the markup now.